config/iam/controller-iam-policy.json

```diff
@@ -0,0 +1,93 @@
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
```

Comment:
A couple thoughts:

1. Can we use managed policies for this App Mesh instead of explicit? This will be easier on both customers and us when we make future changes. It's also safe to simplify this to: 

```
appmesh:Create*
appmesh:Describe*
appmesh:Update*
appmesh:Delete*
appmesh:List*
```

2. For the preview environment these will need to be `appmesh-preview`. We want a separate policy for that in general, since we experiment with things in preview (and wouldn't want to auto add to the prod policy). There are managed policies for preview as well.

Reply:
Thanks for the review. The reason for not adding wildcards in the `appmesh:Create*` etc is because it would add permissions (e.g CreateTags) or any other operations that might be added to AppMesh but the controller.

+1, I will add the preview variant separately